dev/com.ibm.ws.sipcontainer/src/com/ibm/ws/sip/container/tu/TransactionUserBase.java

```diff
@@ -611,19 +611,17 @@ private static String createApplicationId( String srvrid){
 	 * @return
 	 */
 	List<IBMSipSession> getAllSipSessions(List<IBMSipSession> sessions, boolean create){
-		if(_sipSession == null){
-			getSipSession(create);
+		if(create) { //boolean is true, create new SIP sessions if there are none
+			if(_sipSession == null){
+				getSipSession(create);
+				}
 		}
 		if(_sipSession != null){
 			sessions.add(_sipSession);	
 		}
 		return sessions;
 	}
 
-	/**
-	 * Invalidate the Transaction User and Remove it from the TransactionUserTable
-	 *
-	 */
```

Comment:
Unrelated, but was this comment removed on purpose?

Reply:
comment was not removed on purpose, I guess it got deleted somewhere with the several versions of code I tried along the way. I will re-add the comment

Reply:
This comment wasn't added back in?